src/app/(dashboard)/dashboard/challenges/acceptance/[id]/page.tsx

```diff
@@ -0,0 +1,153 @@
+                  </Link>
+                </div>
+              </div>
+
+              <div className='mt-auto w-full border-t border-brand-400' />
+
+              <div className='mt-2 flex items-center'>
+                <Tag className='size-3 text-brand-400' />
+
+                {data?.techs.map((tech) => (
+                  <Typography.P
+                    key={tech}
+                    className={cn(
+                      'relative ml-2 cursor-default text-sm lowercase',
+                      'after:absolute after:bottom-0 after:left-0 after:h-px after:w-full after:origin-bottom-right after:scale-x-0 after:bg-neutral-400 after:transition-transform after:duration-300 after:ease-in-out',
+                      'hover:after:origin-bottom-left hover:after:scale-x-100'
+                    )}
+                  >
+                    {tech}
+                  </Typography.P>
+                ))}
+              </div>
+            </div>
+
+            <div className='col-span-2 flex w-full flex-col items-center justify-between gap-3'>
+              <div className='relative h-full'>
+                <ChallengeLevelBadge
+                  className='absolute left-4 top-4 text-sm'
+                  challengeLevel={data?.challengeLevel as ChallengeLevel}
+                />
+                <Image
+                  className='aspect-square h-full rounded-2xl object-cover'
+                  alt='Quack Visual!'
+                  src='/img/card-template.jpg'
+                  priority
+                  width={700}
+                  height={700}
+                />
+              </div>
+
+              <Button
+                data-testid='accept-challenge-button'
+                className='h-12 w-full rounded-xl tracking-wide'
+              >
+                ACEITAR DESAFIO
+              </Button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className='flex flex-col gap-3'>
+        <span className='flex items-center gap-2 text-brand-500'>
+          <Rocket className='size-7' />
+
+          <Typography.H2 className='text-2xl font-medium'>
+            Continue se desafiando
+          </Typography.H2>
+        </span>
+
+        <div className='grid w-full grid-cols-12 gap-4 rounded-md'>
+          {challenges.map((challenge) => (
+            <Link
+              href={`/dashboard/challenges/acceptance/${challenge.id}`}
+              className={cn(
+                'col-span-6 flex w-full flex-col',
+                challenge.challengeLevel !== data?.challengeLevel && 'hidden',
+                challenge.id === data?.id && 'hidden'
+              )}
+              key={challenge.title + challenge.id}
+            >
+              <ChallengeCard
+                title={challenge.title}
+                techs={challenge.techs}
+                challengeLevel={challenge.challengeLevel}
+                description={challenge.description}
+              />
+            </Link>
+          ))}
```

Comment:
**Only** challenges with the same difficult level should be shown (as specified in Figma ), not all challengs

Reply:
i handle this logic just using css on line 134